packages/app-store/apps.browser.generated.tsx

```diff
@@ -40,6 +40,7 @@ export const EventTypeAddonMap = {
   fathom: dynamic(() => import("./fathom/components/EventTypeAppCardInterface")),
   ga4: dynamic(() => import("./ga4/components/EventTypeAppCardInterface")),
   giphy: dynamic(() => import("./giphy/components/EventTypeAppCardInterface")),
+  "paywong-payments": dynamic(() => import("./paywong-payments/components/EventTypeAppCardInterface")),
```

Comment:
@hariombalhara do we want - instead of _ ?

Reply:
Yeah, I prefer - in pathname, this is going to be the pathname in the link to the app. ChatGPT has a really nice reasoning for this.
<img width="747" alt="Screenshot 2023-04-03 at 10 05 28 AM" src="https://user-images.githubusercontent.com/1780212/229412457-b3a6901f-7a8f-4ccf-858d-4296a5db274c.png">